webapp-django/crashstats/supersearch/static/supersearch/js/socorro/date_filters.js

```diff
@@ -51,10 +51,15 @@ $(function () {
             var value = parseInt(range.slice(0, -1));
             var unit = range.slice(-1);
             var toDate = moment().utc().toDate();
+            var toDateString = moment(toDate).utc().format("MMM DD, YYYY - HH:mm");
             var fromDate = moment().utc().subtract(value, unit).toDate();
+            var fromDateString = moment(fromDate).utc().format("MMM DD, YYYY - HH:mm");
 
-            setDate('to', toDate);
-            setDate('from', fromDate);
+            filters.to.set('minDate', fromDate);
```

Comment:
Removing `setDate` means that the current value of the flatpickr input isn't being set to a different value. If you click the search button after clicking a shortcut link with this code, you'll notice that the date in the URL doesn't match the displayed value because of this.

Setting the date values will also cause the `onChange` handlers above to run, which will remove the need to manually set the `minDate` and `maxDate` here, as well as removing the need to call `removeSelectedShortcut`.

Reply:
I see. Thanks! I will fix it.